third_party/terraform/utils/provider.go.erb

```diff
@@ -226,6 +226,27 @@ func ResourceMapWithErrors() (map[string]*schema.Resource, error) {
 				"google_kms_crypto_key":                        resourceKmsCryptoKey(),
 				"google_kms_crypto_key_iam_binding":            ResourceIamBindingWithImport(IamKmsCryptoKeySchema, NewKmsCryptoKeyIamUpdater, CryptoIdParseFunc),
 				"google_kms_crypto_key_iam_member":             ResourceIamMemberWithImport(IamKmsCryptoKeySchema, NewKmsCryptoKeyIamUpdater, CryptoIdParseFunc),
+<% unless version == 'ga' -%>
+				"google_iap_tunnel_instance_iam_binding":                ResourceIamBindingWithImport(IamIapTunnelInstanceSchema, NewIapTunnelInstanceIamUpdater, IapTunnelInstanceIdParseFunc),
+				"google_iap_tunnel_instance_iam_member":                 ResourceIamMemberWithImport(IamIapTunnelInstanceSchema, NewIapTunnelInstanceIamUpdater, IapTunnelInstanceIdParseFunc),
+				"google_iap_tunnel_instance_iam_policy":                 ResourceIamPolicyWithImport(IamIapTunnelInstanceSchema, NewIapTunnelInstanceIamUpdater, IapTunnelInstanceIdParseFunc),
+<% end -%>
+				"google_logging_billing_account_sink":          resourceLoggingBillingAccountSink(),
```

Comment:
Looks like this block managed to sneak in between revisions; it's just above the added block as well, so one or the other can be removed.